### PR TITLE
added the section for works and frameworks

### PR DESCRIPTION
**Pull request by Ade179**
In this milestone, I have learnt to set up a repo even more professionally and gained more experience while building /completing the mobile version of my portfolio website. A great and tasking experience

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,31 +1,201 @@
...
+        <div class="picfix">
+        <div class="abt">
+            <h3 class="bou-me">About me</h3>
+            <p class="abt-text">Hello I’m a software developer!
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <button class="resume">Get My Resume</button>
+        </div>
```
Comment (tresorsawasawa) [inline index.html:159]: - [ ] kindly make sure the `Get My Resume` button follow the button styling: **Active**, **Hover** and **Pressed** (see design) as from Figma design.
You can find every styling detail on the below image from your chosen **[Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)**:
buttons_2

File: index.html [hunk 2]
```diff
@@ -1,31 +1,201 @@
...
+                no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="skills">
+                <button><li class="js" id="btn">css</li></button>
+                <button><li class="js" id="btn">html</li></button>
+                <button><li class="js" id="btn">javascript</li></button>
+            </ul>
+        </div>
+       <a href=""><button type="button" class="btnsettings">See project</button></a>
```
Comment (Ismailco) [inline index.html:52]: - [ ] [OPTIONAL] Good work using the `<button>/<a>` tag on this link/button however you should use only one so consider showing between  `<button>` or `<a>`

Comment (Ismailco) [inline index.html:52]: - [ ] As the previous reviewer mentioned kindly make sure all `See Project` buttons follow the button styling: Active, Hover, and Pressed (see design) as from Figma design, for now it doesn't

File: index.html [hunk 3]
```diff
@@ -1,31 +1,201 @@
...
+    <section class="About-me">
+        <div class="picfix">
+        <div class="abt">
+            <h3 class="bou-me">About me</h3>
+            <p class="abt-text">Hello I’m a software developer!
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+           <a href=""> <button class="resume">Get My Resume</button></a>
```
Comment (Ismailco) [inline index.html:158]: - [ ] As the previous reviewer mentioned kindly make sure all `See Project` buttons follow the button styling: Active, Hover, and Pressed (see design) as from Figma design, for now, it doesn't change the color back when active
- [ ] [OPTIONAL] Kindly only use a button tag or a tag for this link/button

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Ade179, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that has some failing linter checks. kindly fix your linter errors and submit again 👍🏼 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ichala — APPROVED** [review]
Hi @Ade179,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Good commit messages ✔️
- Linters are passing ✔️
- All Changes from previous reviewers has been made ✔️

Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 
**_this comment is optional but highly recommended _**

- [ ] [OPTIONAL] kindly update your README file using this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) to make it looks professional and descriptive


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**